doc/source/conf.py

```diff
@@ -13,7 +13,8 @@
 from ansys_sphinx_theme import (
     __version__,
     ansys_favicon,
-    ansys_logo_black,
+    ansys_logo_dark,
+    ansys_logo_light,
```

Comment:
Let's go with the approach @klmcadams suggested by renaming these. Even if we break backwards support.